src/formattedcode/writers.py

```diff
@@ -31,16 +31,20 @@
 import os
 from os.path import abspath
 
+import pluggy
```

Comment:
It can be removed, it's left over from an older commit where the post scan plugin manager was instantiated in this file. I'll remove it now.